### PR TITLE
Explain why LRS is expected to use 'exact' when getting statements

### LINKED ISSUE #760: Explain why LRS is expected to use 'exact' when getting statements
The GET Statements table says:

```An LRS requesting Statements for the purpose of importing them would use a format of "exact".``` 

We don't explain this any further anywhere, for example we have no:

* MUST/SHOULD requirements relating to this (maybe it's hard to test)
* Explanation of why it's important
* Explanation of how canonical definitions might be transferred between LRSs

Should we add some/all of those things?

### PR DESCRIPTION
**Pull request by garemoko**
Fix #760

### CODE REVIEW

**Comment by andyjohnson** [comment]
+1

File: xAPI-Communication.md [hunk 1]
```diff
@@ -522,7 +522,8 @@ Object.
 			<br/><br/>
 			If 'exact', return Agent, Activity, Verb and Group Objects populated exactly as they 
 			were when the Statement was received. An LRS requesting Statements for the purpose 
-			of importing them would use a format of 'exact'.  
+			of importing them would use a format of 'exact' in order to maintain 
+			<a href="./xAPI-Data.md#statement-immutablity-and-exceptions">Statement Immutability</a>a>.
```
Comment (TylerDixon) [inline xAPI-Communication.md]: Extra `a>`

**Comment by garemoko** [comment]
@TylerDixon Thanks! Fixed.

**Comment by ljwolford** [comment]
+1

**Pull request merged**